packages/vuetify/index.js

```diff
@@ -22,17 +23,29 @@ module.exports = function (moduleOptions) {
     })
   }
 
+  if (options.treeShake) {
+    const VuetifyLoader = require(path.resolve(this.options.srcDir, 'node_modules', 'vuetify-loader/lib/plugin'))
```

Comment:
This line may introduce some inconsistencies. Would you please try `this.nuxt.resolver.requireModule('vuetify-loader/lib/plugin')`?